sdk/src/main/java/com/kin/ecosystem/base/BaseViewHolder.java

```diff
@@ -90,6 +97,14 @@ protected void setTextColor(@IdRes int viewId, @ColorInt int color) {
 		view.setTextColor(color);
 	}
 
+	protected void setVectorDrawable(@IdRes int viewId, @DrawableRes int vectorResID) {
+		if(!AppCompatDelegate.isCompatVectorFromResourcesEnabled()) {
```

Comment:
what is the minimum android vector drawable works
is it ok with our minimum

Reply:
from what we tested 19+